main_test.go

```diff
@@ -608,7 +553,6 @@ func TestGenerate(t *testing.T) {
     "cluster": "cluster",
     "metadata": {
       "INSTANCE_IP": "10.9.8.7",
-      "TRAFFICDIRECTOR_DIRECTPATH_C2P_IPV6_CAPABLE": true,
```

Comment:
How are there these two (here and just below) changes to the tests? Isn't deleting client_default_listener_resource_name_template clearly a bug?